src/edu/washington/escience/myria/coordinator/MasterCatalog.java

```diff
@@ -129,7 +129,7 @@
           + "    program_name TEXT NOT NULL,\n"
           + "    relation_name TEXT NOT NULL,\n"
           + "    num_shards INTEGER NOT NULL,\n"
-          + "    how_partitioned TEXT NOT NULL,\n"
+          + "    how_distributed TEXT NOT NULL,\n"
```

Comment:
Yes, and we may need to mark some as "unknown" since we have changed the internal representation of this field.